en/08_Changelogs/5.4.0.md

```diff
@@ -15,16 +15,57 @@ title: 5.4.0 (unreleased)
 
 - The class names for the `TopPage` feature in [`dnadesign/silverstripe-elemental`](https://github.com/silverstripe/silverstripe-elemental) do not follow the correct naming convention for Silverstripe CMS. The existing classes have been deprecated and will be renamed to match the correct naming convention in a future major release.
+- [`CliBypass`](api:SilverStripe\Control\Middleware\ConfirmationMiddleware\CliBypass) has been deprecated. It will be removed without equivalent functionality to replace it.
+- [`CliController`](api:SilverStripe\Control\CliController) has been deprecated. It will be replaced with `symfony/console` commands.
+- [`DatabaselessKernel`](api:SilverStripe\Core\DatabaselessKernel) has been deprecated. Use [`CoreKernel::setBootDatabase()`](api:SilverStripe\Core\CoreKernel::setBootDatabase()) instead.
+- [`BuildTask.segment`](api:SilverStripe\Dev\BuildTask->segment) has been deprecated. It will be replaced with a new `$commandName` property.
+- [`BuildTask->description`](api:SilverStripe\Dev\BuildTask->description) has been deprecated. It will be replaced with a new static property with the same name.
+- [`BuildTask::getDescription()`](api:SilverStripe\Dev\BuildTask::getDescription()) has been deprecated. It will be replaced with a new static method with the same name.
+- [`DevBuildController`](api:SilverStripe\Dev\DevBuildController) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\DbBuild` class.
+- [`DevConfigController`](api:SilverStripe\Dev\DevConfigController) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\ConfigDump` class.
+- [`DatabaseAdminDatabaseAdmin`](api:SilverStripe\ORM\DatabaseAdminDatabaseAdmin) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\DbBuild` class.
+- [`DevelopmentAdmin::buildDefaults()`](api:SilverStripe\Dev\DevelopmentAdmin::buildDefaults()) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\DbDefaults` class.
+- [`DevelopmentAdmin::generatesecuretoken()`](api:SilverStripe\Dev\DevelopmentAdmin::generatesecuretoken()) has been deprecated. It will be replaced with a new `SilverStripe\Dev\Command\GenerateSecureToken` class.
+- [`DevelopmentAdmin::getRegisteredController()`](api:SilverStripe\Dev\DevelopmentAdmin::getRegisteredController()) has been deprecated. It will be removed without equivalent functionality to replace it.
+- [`DevelopmentAdmin.registered_controllers`](api:SilverStripe\Dev\DevelopmentAdmin->registered_controllers) has been deprecated. It will be replaced with new `controllers` and `commands` configuration properties.
+- [`CleanupTestDatabasesTask::canView()`](api:SilverStripe\Dev\Tasks\CleanupTestDatabasesTask::canView()) has been deprecated. It will be replaced with a new `canRunInBrowser()` method.
+- [`HTTPOutputHandler`](api:SilverStripe\Logging\HTTPOutputHandler) has been deprecated. It will be renamed to `SilverStripe\Logging\ErrorOutputHandler`
```

Comment:
A little weird that you also have HTTPOutputHandler.isCli() deprecated, which has a replacement - either don't deprecate it or mention it here

Reply:
There's a lot of deprecations here, so forgetting one is understandable I think lol.
Added.